configuration/wireless/wireless-cli.md

```diff
@@ -13,11 +13,11 @@ Select the **Network Options** item from the menu, then the **Wi-fi** option. On
 
```

Comment:
The setting for wireless country code seems to have moved - it's now under 'Localisation Options'. So:

**Network Options** -> **Localisation Options**
**Wi-fi** -> **Change wireless country**